workflow/snakemake_rules/export_for_nextstrain.smk

```diff
@@ -157,18 +157,18 @@ except:
     # means that the Snakefile won't crash.
     deploy_origin = "by an unknown identity"
 
-rule deploy_to_staging:
+rule deploy:

```

Comment:
Sorry, I typed too soon! Since this rule gets renamed, we should update [the reference to `deploy_to_staging` in the trial build config](https://github.com/nextstrain/ncov/pull/629/files#diff-3e38b7b89f28aa3eb17024c02a6ae6a641942711e90a07cbcd155935606384b9R40) and [the corresponding reference in ncov-ingest's rebuild-staging script](https://github.com/nextstrain/ncov-ingest/blob/04ca33cbed1f96320035b9f7ebcc6abf4fa25a72/bin/rebuild-staging#L58).

Reply:
Ah, thank you for that catch. Repushed. I thought I grepped both repos!